src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.ProductDao;
+import core.basesyntax.dao.ProductDaoImp;
+import core.basesyntax.model.ProductTransaction;
+import core.basesyntax.service.ReaderService;
+import core.basesyntax.service.ReportService;
+import core.basesyntax.service.TransactionParser;
+import core.basesyntax.service.WriterService;
+import core.basesyntax.service.imp.ReaderServiceImp;
+import core.basesyntax.service.imp.ReportServiceImp;
+import core.basesyntax.service.imp.TransactionParserImp;
+import core.basesyntax.service.imp.WriterServiceImp;
+import core.basesyntax.strategy.ActivitiesStrategy;
+import core.basesyntax.strategy.ActivitiesStrategyImp;
+import core.basesyntax.strategy.activities.ActivitiesHandler;
+import core.basesyntax.strategy.activities.AddActivitiesHandler;
+import core.basesyntax.strategy.activities.SubstractActivitiesHandler;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final String SOURCE_FILE_NAME = "src/main/resources/DayRecords.csv";
+    private static final String DESTINATION_FILE_NAME = "src/main/resources/Report.csv";
+    private static final String DAY_RECORDS_HEADER = "type,fruit,quantity";
+    private static final String REPORT_HEADER = "fruit,quantity";
+
+    public static void main(String[] args) {
+        ProductDao productDao = new ProductDaoImp();
+        ActivitiesHandler addActivitiesHandler = new AddActivitiesHandler(productDao);
+        ActivitiesHandler subActivitiesHandler = new SubstractActivitiesHandler(productDao);
+
+        Map<ProductTransaction.Operation, ActivitiesHandler> activitiesHandlerMap = new HashMap<>();
+        activitiesHandlerMap.put(ProductTransaction.Operation.BALANCE, addActivitiesHandler);

```

Comment:
I think if we obtain something like BALANCE meaning it doesn't means we should just add it to out quantites.  In my opinion we should set the new quantity (imagine it was inventory or something, and an old quantity doesn't reflect real amount of fruits). In my project I added another one operation, like set to Map. But maybe your solution also respond general condition...